dd-sdk-android/src/main/kotlin/com/datadog/android/error/internal/CrashReportsFeature.kt

```diff
@@ -0,0 +1,85 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2020 Datadog, Inc.
+ */
+
+package com.datadog.android.error.internal
+
+import android.content.Context
+import com.datadog.android.DatadogConfig
+import com.datadog.android.DatadogEndpoint
+import com.datadog.android.core.internal.domain.NoOpPersistenceStrategy
+import com.datadog.android.core.internal.domain.PersistenceStrategy
+import com.datadog.android.core.internal.net.DataUploader
+import com.datadog.android.core.internal.net.NoOpDataUploader
+import com.datadog.android.core.internal.net.info.NetworkInfoProvider
+import com.datadog.android.core.internal.time.TimeProvider
+import com.datadog.android.log.internal.domain.Log
+import com.datadog.android.log.internal.net.LogsOkHttpUploader
+import com.datadog.android.log.internal.user.UserInfoProvider
+import okhttp3.OkHttpClient
+
+internal object CrashReportsFeature {
+
+    internal var originalUncaughtExceptionHandler = Thread.getDefaultUncaughtExceptionHandler()
+    internal var initialized: Boolean = false
+
+    internal var clientToken: String = ""
```

Comment:
I am a bit worried that we are keeping these here as static - mutable vars. Why do you need to keep those here and not taking them from the config and have everything immutable ? The config can be kept as a private val inside.

Reply:
Because this is a static class, the data in there needs to be either a `var` or a `lateininit var`. So even if I kept a ref to the config, that would still be a mutable variable.